### DISCUSSION TITLE
Pydantic v2 support

### DISCUSSION THREAD

**Discussion by Derfirm** [BeanieODM/beanie#597] [Feature Request]
As Pydantic V2 is implemented with Rust, it should be quite performant than V1
https://docs.pydantic.dev/dev-v2/
also wa can re-use utils for migration 
https://github.com/pydantic/bump-pydantic

**Comment by roman-right** [comment]
Hi! Yes, v2 will be supported, but I don't think I'll bump it before the stable version is released. I'm currently working on it using the beta version. However, since I want to support both v1 and v2 in the next few versions of Beanie, I cannot bump to beta. The interfaces may change, and I would have to support too many different interfaces simultaneously.

**Comment by TheKaram** [comment]
Pydantic 2 stable was released three days ago. Have you started working on supporting the new version?
Thank you

**Reply by roman-right** [reply]
Hi @TheKaram,

Yes, I've started. I plan to finish this in 1-2 weeks. Currently, I'm working on bumping Pydantic in the `lazy_model` project, which provides the base class for Beanie documents. You can find the pull request at: https://github.com/roman-right/lazy_model/pull/4

**Reply by TheKaram** [reply]
Great, thank You

**Comment by zhuxining** [comment]
mark an error  #609 

```
from beanie import PydanticObjectId 

......
```
pydantic.error_wrappers:ErrorWrapper has been removed in V2

For further information visit https://errors.pydantic.dev/2.0.2/u/import-error
